deploy/dockerfiles/browser/browser.dockerfile

```diff
@@ -36,6 +34,8 @@ COPY --from=build /home/node/app/browser/dist/public /usr/share/nginx/html
 
 COPY deploy/dockerfiles/browser/browser.nginx.conf /etc/nginx/browser.nginx.conf.template
 
+USER node
```

Comment:
This should be reverted -- since it comes after the `FROM nginx:stable-alpine` line, it'll result in running processes as a user that doesn't exist in the `nginx` image, rather than what the image intends, which runs things as the `nginx` user.